datashader/__init__.py

```diff
@@ -11,6 +13,18 @@
 from . import transfer_functions as tf                   # noqa (API import)
 from . import data_libraries                             # noqa (API import)
 
+with suppress(ImportError):
```

Comment:
This is not the right place it should be in __main__.py.

Reply:
OK!